### DISCUSSION TITLE
Multibyte-character output misalignment with tags

### DISCUSSION THREAD

**Discussion by vladdeSV** [jotaen/klog#350] [Bugs]
If my tag include a multibyte character, when using the command `klog tags`, the tags are not aligned properly.

Assumme we have the file `test.klg`:

```klog
2024-08-01
  1h #a
  1h #b
  1h #ß
```

Running `klog tags test.klg` will produce the output:
```
#a  1h
#b  1h
#ß 1h
```
Here we can see that #a and #b are aligned, but row with the `ß` is missing a spacing character.

If instead of using a `ß` (two bytes) we use `𒀀` (four bytes) we get this output:

```klog
#a    1h
#b    1h
#𒀀 1h
```

<details>
<summary>Finally, we can have a little fun with this</summary>

This file:
```klog
2024-08-01
  1h #ö
  1h #öö
  1h #ööö
  1h #öööö
  1h #ööööö
  1h #öööööö
```
… produces this output:
```
#ö           1h
#öö         1h
#ööö       1h
#öööö     1h
#ööööö   1h
#öööööö 1h
```

</details>

**Comment by jotaen** [comment]
Ah, thanks for reporting this! So far, the underlying table implementation just did a naive byte count, which of course yields incorrect results for multibyte Unicode strings. https://github.com/jotaen/klog/pull/324 fixes this issue.
